activities/zio-flow-twilio/shared/src/main/scala/zio/flow/activities/twilio/CreateMessage.scala

```diff
@@ -0,0 +1,52 @@
+package zio.flow.activities.twilio
+
+import zio.schema._
+import zio.flow._
+
+final case class CreateMessage(
+  To: PhoneNumber,
```

Comment:
We cannot annotate the fields with "schema names" so we have to use the uppercase field names here to make the form encoding matching the specs.

Reply:
🤦‍♂️